### DISCUSSION TITLE
v4.1.0 Release Link Broken

### DISCUSSION THREAD

**Discussion by steve1515** [rtl-airband/RTLSDR-Airband#376] [Q&A] [answered]
It looks like 4.1.0 was recently release and is now the main branch, but when going to the release link listed ([Release Link](https://github.com/charlie-foxtrot/RTLSDR-Airband/releases/latest)), the latest showing is still 4.0.3. I'm guessing that the release wasn't actually created after merging to master.

**Comment by charlie-foxtrot** [comment, marked as answer]
Good catch, fixed
